metricbeat/module/docker/memory/helper.go

```diff
@@ -0,0 +1,56 @@
+package memory
+
+import (
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/metricbeat/module/docker"
+	"github.com/elastic/beats/libbeat/logp"
+)
+
+type MEMORYData struct {
```

Comment:
Is there a reason for the caps lock Memory? If not I would use MemoryData, same  for the implementations below.